server/auth/types/authentication_type.test.ts

```diff
@@ -33,6 +37,9 @@ class DummyAuthType extends AuthenticationType {
   resolveTenant(): Promise<string | undefined> {
     return Promise.resolve('dummy_tenant');
   }
+  public supportsKeepAlive(request: OpenSearchDashboardsRequest): Promise<boolean> {
```

Comment:
Why not just return a boolean here? Is this needed to be async?

Reply:
This needs to be async since the type I defined on the AuthenticationType is async. It was defined to be async since in multi_auth, we need to potentially look at the cookie to figure out what auth is being used:

https://github.com/opensearch-project/security-dashboards-plugin/blob/main/server/auth/types/multiple/multi_auth.ts#L115

That is async, and thus to figure out whether multi_auth supports keepalive needs to be async.

Reply:
Removed this so resolving